src/pages/index.css

```diff
@@ -70,3 +70,10 @@
     padding-right: 0;
   }
 }
+
+.search-results-header {
```

Comment:
can you use [BEM](https://getbem.com/naming/) format: like `search-results__header`